Source/WebCore/animation/AnimationEffect.cpp

```diff
@@ -134,7 +134,7 @@ ComputedEffectTiming AnimationEffect::getComputedTiming(std::optional<WebAnimati
     computedTiming.iterations = m_timing.iterations;
     computedTiming.duration = computedDuration;
     computedTiming.direction = m_timing.direction;
-    computedTiming.easing = m_timing.timingFunction->cssText();
+    computedTiming.easing = m_timing.timingFunction.get()->cssText();
```

Comment:
Ditto. You need to store this value in a stack allocated RefPtr before calling cssText.